src/applications/disability-benefits/4142/helpers.jsx

```diff
@@ -474,18 +466,18 @@ export const stateNames = [
  * Converts the treatment date range into an array of objects from just an object
  * @param {object} treatmentDateRange object containing from/to date range
  */
-const transformDateRange = (treatmentDateRange) => {
+const transformDateRange = treatmentDateRange => {
```

Comment:
This is probably better written utilizing the es6 implicit `return`ing.

`const transformDateRange = treatmentDateRange => ([treatmentDateRange]);`